### ISSUE TITLE
Issue when reading Berkfile - Berkshelf::BerksfileReadError (Berkshelf::BerksfileReadError)

### ISSUE THREAD

**Issue by EmmettKennedy** [lostisland/faraday#420]
Hi Guys, 

Im not sure this belongs in here, if not I apologise. When executing chefspec tests the following error is thrown;
```
/Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/faraday-0.9.0/lib/faraday/connection.rb:310:in `url_prefix=': Berkshelf::BerksfileReadError (Berkshelf::BerksfileReadError)
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/faraday-0.9.0/lib/faraday/connection.rb:77:in `initialize'
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/berkshelf-api-client-1.2.0/lib/berkshelf/api_client/connection.rb:51:in `initialize'
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/berkshelf-api-client-1.2.0/lib/berkshelf/api_client.rb:15:in `new'
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/berkshelf-api-client-1.2.0/lib/berkshelf/api_client.rb:15:in `new'
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/berkshelf-3.1.5/lib/berkshelf/source.rb:13:in `initialize'
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/berkshelf-3.1.5/lib/berkshelf/berksfile.rb:185:in `new'
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/berkshelf-3.1.5/lib/berkshelf/berksfile.rb:185:in `source'
from /Users/ekennedy/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/berkshelf-3.1.5/lib/berkshelf/mixin/dsl_eval.rb:26:in `block (3 levels) in clean_room'
from Berksfile:1:in `block in dsl_eval_file'
```

This is the corresponding entry in the Berksfile

`source 'http://berkshelf-api.acme.com'`

**Comment by kennethkalmer** [comment]
I think this got fixed in berkshelf/berkshelf-api-client@f4ce01ac36b. Regardless, this isn't an issue with Faraday, but in how Berkshelf subclasses `Faraday::Connection`...

/cc @EmmettKennedy @technoweenie

**Comment by technoweenie** [comment]
Thanks for the update!
